e2e/test/scripts/infrastructure-linode/0.1.0/infrastructure-components.yaml

```diff
@@ -0,0 +1,1749 @@
+    app.kubernetes.io/name: clusterrolebinding
+    app.kubernetes.io/part-of: cluster-api-provider-linode
+  name: capl-manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: capl-manager-role
+subjects:
+- kind: ServiceAccount
+  name: capl-controller-manager
+  namespace: capl-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app.kubernetes.io/component: kube-rbac-proxy
+    app.kubernetes.io/created-by: cluster-api-provider-linode
+    app.kubernetes.io/instance: proxy-rolebinding
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: clusterrolebinding
+    app.kubernetes.io/part-of: cluster-api-provider-linode
+  name: capl-proxy-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: capl-proxy-role
+subjects:
+- kind: ServiceAccount
+  name: capl-controller-manager
+  namespace: capl-system
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: capl-manager-credentials
+  namespace: capl-system
+stringData:
+  apiToken: ${LINODE_TOKEN}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/component: kube-rbac-proxy
+    app.kubernetes.io/created-by: cluster-api-provider-linode
+    app.kubernetes.io/instance: controller-manager-metrics-service
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: service
+    app.kubernetes.io/part-of: cluster-api-provider-linode
+    control-plane: controller-manager
+  name: capl-controller-manager-metrics-service
+  namespace: capl-system
+spec:
+  ports:
+  - name: https
+    port: 8443
+    protocol: TCP
+    targetPort: https
+  selector:
+    control-plane: controller-manager
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: cluster-api-provider-linode
+    app.kubernetes.io/instance: controller-manager
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: deployment
+    app.kubernetes.io/part-of: cluster-api-provider-linode
+    control-plane: controller-manager
+  name: capl-controller-manager
+  namespace: capl-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+      labels:
+        control-plane: controller-manager
+    spec:
+      containers:
+      - args:
+        - --health-probe-bind-address=:8081
+        - --metrics-bind-address=127.0.0.1:8080
+        - --leader-elect
+        command:
+        - /manager
+        env:
+        - name: LINODE_TOKEN
+          valueFrom:
+            secretKeyRef:
+              key: apiToken
+              name: capl-manager-credentials
+        - name: LINODE_API_VERSION
+          value: v4beta
+        image: docker.io/linode/cluster-api-provider-linode:main
```

Comment:
Linode CAPL provider doesn't have any official releases, so this image doesn't exist! This PR has to wait until the provider has been released.

Reply:
@mhmxs the first alpha release of CAPL has been deployed so you can now use the :latest image which will give you the latest versioned image